modules/openapi-generator/src/main/resources/Java/libraries/okhttp-gson/pojo.mustache

```diff
@@ -13,7 +13,6 @@ import com.google.gson.stream.JsonReader;
 import com.google.gson.stream.JsonWriter;
 import java.io.IOException;
 
-import java.lang.reflect.Type;
```

Comment:
don't recall the exact reason why it's added in the first place.

if this breaks any other use cases somehow, we will need to roll back this fix and use model name mapping option instead: https://github.com/openapitools/openapi-generator/blob/master/docs/customization.md#name-mapping